assets/src/containers/ResourcesAccessed.js

```diff
@@ -236,7 +229,6 @@ function ResourcesAccessed (props) {
       )
     }
   }
-  if (error) return (<WarningBanner/>)
```

Comment:
Removed this as it is not needed, when we put the content call in `api.js` then we can handle this. The BE always returns empty response so logic is baked in to display appropriate message to user